### DISCUSSION TITLE
Ubuntu Docker image missing for 5.3.2

### DISCUSSION THREAD

**Discussion by CMircea** [ravendb/ravendb#13475] [Q&A] [answered]
Hello!

I have an alert that 5.3.2 is available, but the latest Ubuntu Docker image is for 5.3.1; only Windows has 5.3.2. Was this overlooked, or does 5.3.2 (with the transaction rollback w/ compression fix) affect only Windows?

Thanks!

Mircea

**Comment by ayende** [comment]
Likely an issue on the deployment, yes. We'll deal with it this week

**Reply by CMircea** [reply]
Thanks!

**Comment by gregolsky** [comment, marked as answer]
Images have been pushed. Sorry for the inconvenience.
![image](https://user-images.githubusercontent.com/1541805/149732012-100daa72-2d64-47d4-a9ec-69d45e9dba81.png)

**Reply by CMircea** [reply]
No worries, thank you!
